app/shared/utils/Table.js

```diff
@@ -92,149 +62,135 @@ export default function Table({
     [filterColumns],
   )
 
-  const initialFilters = useMemo(() => defaultFilters, [])
-
   const initialState = useMemo(
     () => ({
-      pageIndex: 0,
-      sortBy: initialSortById ? [{ id: initialSortById, desc: true }] : [],
-      filters: initialFilters,
+      pagination: {
+        pageIndex: 0,
+        pageSize: pagination ? 10 : 10000,
+      },
+      sorting: initialSortById ? [{ id: initialSortById, desc: true }] : [],
+      filters: defaultFilters,
     }),
-    [initialSortById],
-  )
-
-  const {
-    getTableProps,
-    getTableBodyProps,
-    headerGroups,
-    prepareRow,
-    page,
-    canPreviousPage,
-    canNextPage,
-    pageOptions,
-    pageCount,
-    gotoPage,
-    nextPage,
-    previousPage,
-    setPageSize,
-    state: { pageIndex, pageSize },
-  } = useTable(
-    {
-      columns,
-      data,
-      initialState,
-      defaultColumn,
-      filterTypes,
-    },
-    useFilters,
-    useGlobalFilter,
-    useSortBy,
-    usePagination,
+    [initialSortById, pagination],
```

Comment:
Something's off here.. If I pull your branch and try to paginate on the table in the admin area, it doesn't work, and the request that's generated ends up being a _websocket_ request for some reason.

If I switch over to `develop` everything is working fine.

Reply:
now the page just locks up for me: https://goodpartyorg.slack.com/archives/C06NQMW4HJQ/p1747085233005849